deploy/templates/nstemplatetiers/test/ns_dev.yaml

```
@@ -13,6 +13,61 @@ objects:
     labels:
       name: ${USERNAME}-dev
     name: ${USERNAME}-dev
+
+  # Quotas and default limits for not-terminating containers (regular long-running containers)
+  # and terminating (short-lived containers like build) containers
+- apiVersion: v1
+  kind: ResourceQuota
+  metadata:
+    name: compute-deploy
+    namespace: ${USERNAME}-dev
+  spec:
+    scopes:
+      - NotTerminating
+    hard:
+      limits.cpu: "20"
```

```suggestion
      limits.cpu: 20000m
```
It's the same but it's easier to test in e2e tests if it's defined in milli-cores (easier to compare strings in m).